eventmesh-connectors/eventmesh-connector-file/src/main/resources/source-config.yml

```diff
@@ -15,20 +15,13 @@
 # limitations under the License.
 #
 
-pubSubConfig:
-    meshAddress: 127.0.0.1:10000
-    subject: TopicTest
-    idc: FT
-    env: PRD
-    group: fileSource
-    appId: 5032
-    userName: fileSourceUser
-    passWord: filePassWord
 connectorConfig:
     connectorName: fileSource
     nameserver: 127.0.0.1:9877
     topic: TopicTest
     commitOffsetIntervalMs: 5000
+    fileName: TopicTest-%s
+    filePath: TopicTest/%s
```

Comment:
LGTM. May you please provide a config document [here](https://github.com/apache/eventmesh-site/tree/master/docs/design-document/03-connect) later?